lang/src/org/partiql/lang/eval/CompileOptions.kt

```diff
@@ -63,7 +64,8 @@ enum class RewritingMode {
 data class CompileOptions private constructor (
         val undefinedVariable: UndefinedVariableBehavior,
         val projectionIteration: ProjectionIterationBehavior = ProjectionIterationBehavior.FILTER_MISSING,
-        val rewritingMode: RewritingMode = RewritingMode.DEFAULT
+        val rewritingMode: RewritingMode = RewritingMode.DEFAULT,
+        val thunkOptions: ThunkOptions = ThunkOptions.build{}
```

Comment:
`ThunkOptions.build{}` works but perhaps `ThunkOptions.standard()` is a little better.